src/components/Consent.js

```diff
@@ -114,43 +115,37 @@ const Consent = (props) => {
               <FormControlLabel
                 value="opt-in"
                 checked={value === "opt-in"}
-                inputRef={register}
+
                 className={classes.label}
-                control={<Radio color="primary" required />}
+                control={<Radio color="primary" {...register("privacy")} required />}
                 label={t("consent.opt-in", { partner: config.organisation })}
               />
             )}
             {config.component?.consent?.split && (
               <>
                 <FormControlLabel
                   value="opt-in"
-                  inputRef={register}
                   className={classes.label}
                   checked={value === "opt-in"}
-                  control={<Radio color="primary" required />}
+                  control={<Radio color="primary"  {...register("privacy") } required />}
                   label={t("consent.opt-in", { partner: config.organisation })}
                 />
                 <FormControlLabel
                   value="opt-in-both"
-                  inputRef={register}
                   className={classes.label}
-                  control={<Radio color="primary" />}
+                  control={<Radio {...register("privacy")} color="primary" />}
                   label={t("consent.opt-in-both", {
                     lead: config.lead.title,
                     partner: config.organisation,
                   })}
                 />
               </>
             )}
-
             <FormControlLabel
               value="opt-out"
               checked={value === "opt-out"}
-              control={<Radio />}
+              control={<Radio {...register("privacy")}  required/>}
```

Comment:
is this taking the translation? required here is at the MUI level, not anymore at the RHF level, for instance it doesn't use the translated consent required string, does it?

Reply:
If you mean, if consent is displayed in the chosen language, then yes.
de widget:
![sc](https://user-images.githubusercontent.com/65791349/200264037-3c7ed10e-7c39-43af-86a4-a95cc270b58c.png)
hr widget:
![sc1](https://user-images.githubusercontent.com/65791349/200264045-b54c8ea2-37d4-4291-a8f1-e74f79b9b3d2.png)

Reply:
sorry, I meant the error message if you are trying to submit without selecting a consent

Reply:
This is what is happening now, what is expected?
![sc](https://user-images.githubusercontent.com/65791349/200280846-72112f53-3ee7-4275-b79e-bd1687574c91.png)

Reply:
nvm, I found the error msg. this needs fixing